index.md

```diff
@@ -1,4 +1,125 @@
-# This is the **HOMEPAGE**.
-Refer to [Markdown](http://daringfireball.net/projects/markdown/) for how to write markdown files.
-## Quick Start Notes:
-1. Add images to the *images* folder if the file is referencing an image.
+---
+title: Welcome to Blazor Docs website!
+---
+<div class="hero">
+  <div class="wrap">
+    <div class="text">
+      <strong>BLAZOR</strong>
+    </div>
+    <div class="buttons-unit-small">
+      <a class="version-link" href="#">Version Notes</a><span>|</span><a class="github-link" href="https://github.com/dotnet/docfx">View in Github</a>
+    </div>
+    <div class="minitext">
+        Blazor is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly
+    </div>
+    <div class="buttons-unit">
+      <a href="#" class="button"><i class="glyphicon glyphicon-send"></i>Get Started</a>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-wrench"></i>
+        <section>
+          <h2>Build Web UI with C#</h2>
+          <p class="lead">Blazor is a web UI framework, similar to existing JavaScript frameworks like Angular or React, but you write C# instead of JavaScript.</p>
+          <a href="https://github.com/aspnet/Blazor/wiki/FAQ" class="btn btn-primary">What is Blazor</a>
+        </section>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="counter-key-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <section>
+          <h2>Full Stack .NET</h2>
+          <p class="lead">Do full stack .NET development using stable and consistent tools, languages and APIs both in the browser and on the server. </p>
+          <a href="https://www.microsoft.com/net" class="btn btn-primary">Learn more about the .NET platform</a>
+        </section>
+        <i class="glyphicon glyphicon-tasks"></i>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="key-section">
+  <div class="container content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <i class="glyphicon glyphicon-globe"></i>
+        <section>
+          <h2>.NET Standard, Works in all browsers</h2>
+          <p class="lead">Blazor runs in all browsers, on the real .NET runtime with full support for .NET Standard (i) [info bubble that explains what .NET Standard is]. Blazor requires no plugins and no code transpilation, only open web standards. </p>
```

Comment:
* Remove comma before "on."
* Is "real" necessary?
* Extra space inside the closing `</p>` tag.

Reply:
I'm leaving "real" in. I think the meaning is that this is "really" .NET, not some tiny subset or strange version. I'd use the word "full .NET runtime" if "full framework" wasn't an overloaded term.